### PR TITLE
Don't exit if fail occurred

### PR DESCRIPTION
**Pull request by cupnes**
The following implementation of #25 was missing, so I did it.

> Do not exit when some error occurs

### CODE REVIEW

**Review by toshipp — DISMISSED** [review]
LGTM

**Comment by toshipp** [comment]
oops. these lines should be removed. https://github.com/topolvm/pvc-autoresizer/pull/57/files#diff-eb22bf027de736dd0c2c2eca6857d21450ade485170f674aa713524cbfcda640R224-R227
https://github.com/topolvm/pvc-autoresizer/pull/57/files#diff-eb22bf027de736dd0c2c2eca6857d21450ade485170f674aa713524cbfcda640R246-R249

**Review by toshipp — APPROVED** [review]

File: metrics/resizer.go [hunk 1]
```diff
@@ -41,31 +41,31 @@ func (a *loopSecondsTotalAdapter) Add(value float64) {
...
 		Name:      ResizerLoopSecondsTotalKey,
 		Help:      "counter that indicates the sum of seconds spent on volume expansion processing loops.",
 	})
 
-	ResizerSuccessLoopTotal *resizerSuccessLoopTotalAdapter = &resizerSuccessLoopTotalAdapter{metric: resizerSuccessLoopTotal}
-	ResizerFailedLoopTotal  *resizerFailedLoopTotalAdapter  = &resizerFailedLoopTotalAdapter{metric: resizerFailedLoopTotal}
-	ResizerLoopSecondsTotal *loopSecondsTotalAdapter        = &loopSecondsTotalAdapter{metric: resizerLoopSecondsTotal}
+	ResizerSuccessResizeTotal *resizerSuccessResizeTotalAdapter = &resizerSuccessResizeTotalAdapter{metric: resizerSuccessResizeTotal}
+	ResizerFailedResizeTotal  *resizerFailedResizeTotalAdapter  = &resizerFailedResizeTotalAdapter{metric: resizerFailedResizeTotal}
+	ResizerLoopSecondsTotal   *loopSecondsTotalAdapter          = &loopSecondsTotalAdapter{metric: resizerLoopSecondsTotal}
```
Comment (llamerada-jp) [inline metrics/resizer.go:64]: Is type of loopSecondsTotalAdapter named with different rule from resizerSuccessResizeTotalAdapter and resizerFailedResizeTotalAdapter ?

**Review by toshipp — APPROVED** [review]

**Review by llamerada-jp — APPROVED** [review]
LGTM

**Pull request merged**